### PR TITLE
Make event card color scheme consistent

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/5109

image

image

### CODE REVIEW

File: components/collective-page/sections/Events.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ class SectionEvents extends React.PureComponent {
                 collective={collective}
                 event={event}
                 hideContributors={hasNoContributorForEvents}
-                disableCTA={!collective.isActive || !event.isActive}
```
Comment (Betree) [inline components/collective-page/sections/Events.js:89]: I don't get why we need to set this here, since `ContributeEvent` already do the same check?

Comment (SudharakaP) [reply components/collective-page/sections/Events.js:89]: Ops, yes you are correct, this prop isn't used and I've removed it. 👍🏽

**Review by Betree — COMMENTED** [review]
Have you tested the contribute page also (`/babel/contribute` or `/babel/events`)?

**Comment by SudharakaP** [comment]
> Have you tested the contribute page also (`/babel/contribute` or `/babel/events`)?

@Betree : Thanks for mentioning this. I've checked `<collective-slug>/events` but forgot to check the `<collective-slug>/contribute`. But I've just checked and seems to work fine as well. 👍🏽

**Review by Betree — APPROVED** [review]

**Pull request merged**